### ISSUE TITLE
Remove "Confirm Navigation" (backspace blocker) dialog

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#2760]
We need to always silently block the default action of **backspace/delete** to go back to the previous page in history.

Currently, we pop up a dialog, asking the user if they meant to go back in history, or if they would like to remain on the current page.  This is an unnecessary dialog and we shouldn't bother the user with the query at all.

To show dialog:
1. Open HTML page in VE.
2. In Widgets Palette, click on one of the categories.
3. Hit backspace/delete.

@JonFerraiolo

**Comment by jhpedemonte** [comment]
-> @doronrosenberg

**Comment by doronrosenberg** [comment]
So you want us to never allow the back action then?

**Comment by jhpedemonte** [comment]
No, going back in history by hitting BACKSPACE is a stupid, horrible "feature", often leading to frustration when you realize that your focus wasn't really on that text area you just spent 10 minutes typing into, resulting in you going back and losing your work.

Currently, we instead show a dialog asking the user if they really meant to go back in history when they hit backspace.  I would wager that only about 1 in 1000 users (probably less) would actually use backspace that way.

Let's get rid of the alert and just ignore backspace.

**Comment by JonFerraiolo** [comment]
I agree with what @jhpedemonte says. If users complain, we can address the complaints, but let's try what he proposes for this release.

**Comment by doronrosenberg** [comment]
fixed by 67d5fdac1844e558bd2cceb05635961e53e19f03

**Comment by jhpedemonte** [comment]
Regarding @doronrosenberg's commit: I think we're already handling backspace somewhere (and displaying the alert).  Would need to get rid of that code.

**Comment by aerwin** [comment]
@doronrosenberg 

Reopening... I can no longer delete text using the delete key on my Mac. For example, drag GridX onto page and try to delete characters in the text box on first panel of GridX wizard. I suspect this was caused by the earlier commit here.

**Comment by doronrosenberg** [comment]
@aerwin probably is, looking into it

**Comment by doronrosenberg** [comment]
@jhpedemonte The existing code needs to stay if you press back and we warn if things are unsaved, etc.

Also, backing this out.  There is no good way to do this correctly as far as I can tell without hacking up checks for inputs/etc and even then I bet we will break something else.

**Comment by jhpedemonte** [comment]
Curses!  Foiled again!

We currently have code that detects backspace and puts up a dialog.  Can't we just not show the dialog and absorb the default action?

Pushing to M8, given risk...

**Comment by doronrosenberg** [comment]
The default action is to go back - we could do that, but I would think that most uses of backspace would be user error and data loss could happen.

**Comment by aerwin** [comment]
I had been in that code a long time ago... my recollection is that we catch the unload event to put up the dialog. We don't look for a key press.
